setup_environment.md

```diff
@@ -0,0 +1,45 @@
+# Set up the conda environment for STEMMUS-SCOPE
+
+In this guide, you will learn how to configure the conda environment in your jupyter-workspace on SURF research cloud for running the STEMMUS-SCOPE model in the [notebook](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/blob/main/docs/notebooks/run_model_in_notebook.ipynb).
+
+This guide assumes you already have access to SRC as a user and the workspace created by the administrator of EcoExtreML project group.
+
+In order to run STEMMUS-SCOPE on SRC, we need to configure the `conda` environment and download the executable file.
+
+## Managing environments and kernels on this Jupyter-workspace using `conda`
+In our jupyter-workspace, `miniconda` is pre-installed at the location `/etc/miniconda`. We will use it to create virtual python environments and install [`pystemmusscope`](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing).
+
+To set up the `conda` environment:
+
+- Start a "Terminal" tab in the Jupyter Lab launcher and type:
+  ```py
+  /etc/miniconda/bin/conda init
+  ```
+- Close the terminal tab and start a new one. You will see that the terminal prompt has changed to something like:
+  ```py
+  (base) metheuser@mywsp:
+  ```
+  (When you log in to your jupyter-workspace for the first time, a notebook called "KERNEL-README.ipynb" will be created by default and it is accessbile from files panel. In this notebook, you can find the guidelines about how to manage environments and kernels on this Jupyter-workspace.)
```

Comment:
I logged into the ecoextremlvm workspace, did the conda init, logged in again with SSH and JupyterHub, but did not see KERNEL-README.ipynb file.